readme.md

```diff
@@ -37,6 +38,29 @@ This project is a Discord bot built with `discord.js`. The bot includes various
    - **DISCORD_CLIENT_ID**: Your bot's client ID.
    - **DISCORD_GUILD_ID**: The ID of the server (guild) where the commands will be deployed.
 
+To obtain each of these environment variables, you need to have `Developer Mode` enabled in Discord (if it is not already).
+
+Open Discord to enable developer mode and click on the gear icon (User Settings) at the bottom left. Go to `App Settings` and select the `Advanced` option, then look for the `Developer Mode` option and toggle it on. For more details on how to create a Discord bot, you can visit the official Discord guide.
+
+Once `Developer Mode` is enabled, you can follow these steps to obtain Environment Variables:
+
+### 1. Obtain **DISCORD_TOKEN**
+- Go to the [Discord Developer Portal](https://discord.com/developers/applications) and log in.
+- Create a new application by clicking on “New Application”.
+- Name your application and click "Create".
+- In the "Bot" section, click on "Add Bot" and confirm.
+- Copy the bot token that is generated. This is your **DISCORD_TOKEN**.
+
+### 2. Obtain **DISCORD_CLIENT_ID**
+- In the Discord Portal, select your application (your user avatar).
+- You will find the option to copy the Client ID. This is your **DISCORD_CLIENT_ID**.
```

Comment:
please make the clarification here that it is the same that application ID

Reply:
oh no, this is your app that´s because is Discord portal and not DDP.
This will be your avatar (application):
>![image](https://github.com/user-attachments/assets/21afebc0-bb48-48be-a0b1-a14ea453bcb1)

Reply:
This is what I see when click on my avatar. Should be there the Copy CLIENT_ID option?
![image](https://github.com/user-attachments/assets/2095479e-1da9-4775-9d1f-835c019b1a20)

Just in case, I have Developer mode enabled

Reply:
Oh I didn't know I can use my User ID instead. What is the difference between using User ID instead of Application ID?. Should be this on the documentation?

Reply:
I think that will be the bot token because is from your created App

Reply:
Tried but received this error, it seems like it needs to be a application ID actually. Do it worked for you with the user ID?
![image](https://github.com/user-attachments/assets/b7c7c732-d8d6-4469-802e-1064574cb549)